test/e2e/tests/tokens/nft/send-nft.spec.js

```diff
@@ -145,8 +145,6 @@ describe('Send NFT', function () {
         // Go back to NFTs tab and check the imported NFT is shown as previously owned
         await driver.clickElement('[data-testid="account-overview__nfts-tab"]');
 
-        await driver.clickElement('[data-testid="refresh-list-button"]');
-
```

Comment:
No need to click on refresh button because this Tx is safetransferfrom and ownership has been already refreshed in MM controller